wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
The Array{UInt8,1} is initialized with `0x00`, so yes, null terminated if `OutType` is smaller than the size of the buffer.

I made the patch (I kept the function with only one argument in case someone would prefer it, overloading is working well in Julia).
However, during the testing, I tend to have the first call going well, but not the next, telling that the parameter is invalid (whereas it is valid as it also works sometimes).